integration-tests/jail/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM baiduxlab/sgx-rust:1804-1.0.9
+LABEL maintainer="Crypto.com"
+
+RUN echo 'source /opt/sgxsdk/environment' >> /root/.docker_bashrc 
+RUN echo 'source /root/.cargo/env' >> /root/.docker_bashrc
+RUN apt-get update 
+RUN apt-get install -y --no-install-recommends libzmq3-dev rsync 
+RUN rm -rf /var/lib/apt/lists/*
+RUN apt update
+RUN apt install whois wget curl -y
+RUN apt install python3 libssl-dev libcurl4-openssl-dev libelf-dev libdw-dev  gcc binutils-dev libc6-dev -y
+RUN apt install pkg-config -y
+RUN apt install build-essential openssh-server -y
+RUN apt-get install cmake libgflags-dev libzmq3-dev pkg-config -y
+RUN apt-get install --no-install-recommends libssl-dev libzmq3-dev -y
+RUN apt install unzip -y
+RUN apt install tmux -y
```

Comment:
I suggest to aggregate installation into one line to reduce layers. Also standardize the installation step to `apt install`

Reply:
ok